code/game/objects/items/weapons/stunbaton.dm

```diff
@@ -94,6 +94,36 @@
 			return
 		..()
 
+/obj/item/weapon/melee/baton/attackby(var/obj/item/I, mob/user as mob)
+	..()
+	if(istype(I, /obj/item/weapon/bikehorn))
+
+		if (src.type == /obj/item/weapon/melee/baton/cattleprod)
+			return
+
+		user.visible_message("<span class='notice'>[user] starts jamming \a [src] into the mouth of \a [I].</span>",\
+		"<span class='info'>You do your best to jam \the [src] into the mouth of \the [I].</span>")
+
+		if(do_after(user, get_turf(src), 5 SECONDS))
```

Comment:
This is literal copypaste shit from the thing for making a spear from a wired rod, so should probably change that as well.